test/createProcessor.test.ts

```diff
@@ -1,17 +1,10 @@
+import { equal, rejects } from "node:assert/strict";
 import { describe, test } from "mocha";
 import { assert } from "sinon";
-import { createProcessor, symetricEncrypt } from "../src/main";
-import fetch, { Headers, Request, Response } from "node-fetch";
```

Comment:
I removed node-fetch since the versions of node we are using for tests done need it anymore.